src/qml/components/Separator.qml

```diff
@@ -0,0 +1,12 @@
+// Copyright (c) 2023 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import "../controls"
+
+Rectangle {
+    height: 1
+    color: Theme.color.neutral5
+}
```

Comment:
github complaining about no newline at eof

Reply:
thanks, addressed in 5cbf67f1c479f08cfe3954c56c9962029e6bd5aa